plugins/git/functions

```diff
@@ -33,7 +35,16 @@ git_build_app_repo() {
   local TMP_TAG="dokku/$REV"
   chmod 755 "$GIT_BUILD_APP_REPO_TMP_WORK_DIR"
   unset GIT_DIR GIT_WORK_TREE
-  [[ ! -d "$DOKKU_ROOT/$APP" ]] && apps_create "$APP"
+
+  DOKKU_GLOBAL_DISABLE_AUTOCREATE="$(config_get --global DOKKU_DISABLE_APP_AUTOCREATION)"
```

Comment:
Should be

```
DOKKU_GLOBAL_DISABLE_AUTOCREATE="$(config_get --global DOKKU_DISABLE_APP_AUTOCREATION || true)"
```

Thats why tests fail.

Reply:
I believe the default is intended to be false (discussed in #2808), and the condition further down (L41) explicitly checks for `true`. Is an `|| false` required?

Reply:
The `|| true` does not output true, it merely ensures the exit code is always zero.